### PR TITLE
lock yarn to a specific version

### LINKED ISSUE #4493: Lock yarn to a specific version
**What feature are you requesting?**
Use `yarn policies set-version` to enforce a single version of Yarn across the monorepo.

**What would be the benefit of adding this feature?**
This would ensure developers are all using the same version of Yarn while working within the project.

**What solution would be ideal for you?**

1. Add an `engines` property to the root-level `package.json` to ensure that everyone has at least yarn version 1.13 (the first version to support local instances):
    ```json
    "engines": {
        "yarn": ">=1.13 <2"
    }
    ```

2. Run `yarn policies set-version` in the root directory. This will download the latest 1.x version of Yarn and place it in the `.yarn/releases` directory. This file and directories should be committed and pushed.

3. Add this line to the root-level `.gitattributes` file to prevent Github from including the Yarn source as part of the [language breakdown graph](https://github.com/github/linguist/blob/master/docs/overrides.md):
    ```gitattributes
    .yarn/releases/*.cjs linguist-vendored
    ```

**Is there any additional context for your request?**
These changes should only be made for the root-level monorepo and should not be replicated within the individual packages. **The `engines` field in particular should only be needed in the main `package.json` since we don't want it to be a _directive_ as a downstream dependency.**

Developers will still need a global instance of `yarn`. The global instance automatically checks the `.yarnrc` file and switches to use the local version referenced by the `yarn-path` field.

* `yarn policies` command: https://classic.yarnpkg.com/en/docs/cli/policies
* `yarn-path` field for `.yarnrc`: https://classic.yarnpkg.com/en/docs/yarnrc#toc-yarn-path
* `engines` field for `package.json`: https://classic.yarnpkg.com/en/docs/package-json#toc-engines

### PR DESCRIPTION
**Pull request by chrisdholt**
# Pull Request

## 📖 Description
Implements a single version policy for yarn for development.

closes #4493 

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

**Review by EisenbergEffect — APPROVED** [review]

**Review by scomea — APPROVED** [review]

File: .gitattributes [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # Set the default behavior, in case contributors don't have core.autocrlf set.
 * text=auto
+
+# prevent Github from including yarn source
+.yarn/releases/*.cjs linguist-vendored
```
Comment (radium-v) [inline .gitattributes:5]: The file added is `.yarn/releases/yarn-1.22.11.js` but this is referencing `.yarn/releases/*.cjs`.

**Review by radium-v — CHANGES_REQUESTED** [review]
When I run `yarn policies set version 1` it downloads the file as `yarn-1.22.11.cjs`. This might be due to the version of Node used when installing (I'm on 14.17.5), though I'm not sure.

File: .gitattributes (hunk 1 above)
Comment (chrisdholt) [reply .gitattributes:5]: I went based on your suggestion. I'll edit to add both. As a heads up, I'm using the same and seeing the same. Probably best to support both.
image

**Comment by chrisdholt** [comment]
> When I run `yarn policies set version 1` it downloads the file as `yarn-1.22.11.cjs`. This might be due to the version of Node used when installing (I'm on 14.17.5), though I'm not sure.

Due to multiple versions of yarn existing and the primary global being outdated. Fixed and pushed.

**Review by radium-v — APPROVED** [review]

**Pull request merged**